secure-api-gateway-ob-uk-common-error/src/main/java/com/forgerock/sapi/gateway/ob/uk/common/error/OBRIErrorType.java

```diff
@@ -733,6 +733,10 @@ public enum OBRIErrorType {
             HttpStatus.BAD_REQUEST,
             OBStandardErrorCodes1.UK_OBIE_RULES_FAILS_CONTROL_PARAMETERS,
             "The field '%s' breaches a limitation set by '%s'"),
+    REQUEST_VRP_CONTROL_PARAMETERS_RULES_GENERIC_FAILURE(
+            HttpStatus.BAD_REQUEST,
+            OBStandardErrorCodes1.UK_OBIE_RULES_FAILS_CONTROL_PARAMETERS,
+            "Invalid value provided for field '%s'"),
```

Comment:
Can you improve this error message please? It is a little too generic at the moment.

Can you include the current value and the acceptable value(s).